### DISCUSSION TITLE
pi-Base VS Code extension

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#654] [Show and tell]
@jamesdabbs has built a VS Code extension to help with previewing as you edit the data repository. It's available in full Codespaces as well as in GitHub.dev:

![image](https://github.com/pi-base/data/assets/1559632/345851cf-eb4c-4252-aa25-878ed4758749)

I *think* I've set it up to be installed by default when using GitHub.dev, or at least for it to pop up as a suggestion. Let us know your feedback.

**Comment by prabau** [comment]
The file `.vscode/extensions.json` has the wrong line endings (CRLF).  Can you fix it?

**Reply by StevenClontz** [reply]
Done, thanks

**Reply by StevenClontz** [reply]
And I think I've set up `.vscode/settings.json` to fix this problem by setting line endings to `LF` by default when using `GitHub.dev`.
